persistence/persistence-mybatis/src/main/resources/org/cbioportal/persistence/mybatis/GenePanelMapper.xml

```diff
@@ -103,17 +113,21 @@
         </association>
     </resultMap>
 
-    <select id="getGenePanelBySampleIdAndProfileId" resultType="String">
+    <select id="getGenePanelsByProfile" resultMap="genePanelSampleResultMap">
         select
-            gene_panel.STABLE_ID as stableId
-        from sample_profile
+            gene_panel.STABLE_ID as stableId,
+            gene_panel.DESCRIPTION as description,
+            gene.HUGO_GENE_SYMBOL as hugoGeneSymbol,
+            sample.STABLE_ID as sampleId
+        from genetic_profile
+            inner join sample_profile on genetic_profile.GENETIC_PROFILE_ID = sample_profile.GENETIC_PROFILE_ID
             inner join gene_panel on sample_profile.PANEL_ID = gene_panel.INTERNAL_ID
```

Comment:
If we wanted to get out records which show samples that have no assigned gene panel (whole exon sequenced) we could make this inner join into a left join.  (LEFT JOIN gene_panel on ...)